### PR TITLE
WebGLBackground: recompile background material when tone mapping changes

### PR DESCRIPTION
**Pull request by WestLangley**
As suggested by @mrdoob in https://github.com/mrdoob/three.js/pull/18330#issuecomment-573401941.

### CODE REVIEW

**Comment by WestLangley** [comment]
@mrdoob Implemented, as you requested.

File: src/renderers/webgl/WebGLBackground.js [hunk 1]
```diff
@@ -108,12 +109,14 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 			boxMesh.material.uniforms.flipEnvMap.value = texture.isCubeTexture ? - 1 : 1;
 
 			if ( currentBackground !== background ||
-			     currentBackgroundVersion !== texture.version ) {
+				currentBackgroundVersion !== texture.version ||
+				currentTonemapping != renderer.toneMapping ) {
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLBackground.js]: Maybe `!==` instead of `!=`?

File: src/renderers/webgl/WebGLBackground.js [hunk 2]
```diff
@@ -166,12 +169,14 @@ function WebGLBackground( renderer, state, objects, premultipliedAlpha ) {
 			planeMesh.material.uniforms.uvTransform.value.copy( background.matrix );
 
 			if ( currentBackground !== background ||
-				   currentBackgroundVersion !== background.version ) {
+				currentBackgroundVersion !== background.version ||
+				currentTonemapping != renderer.toneMapping ) {
```
Comment (Mugen87) [inline src/renderers/webgl/WebGLBackground.js]: Same here.

File: src/renderers/webgl/WebGLBackground.js (hunk 1 above)
Comment (WestLangley) [reply src/renderers/webgl/WebGLBackground.js]: Oops. Thanks for the review. I'll fix it.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**